RadixWallet/Core/DesignSystem/Components/PoolUnitView.swift

```diff
@@ -35,8 +37,11 @@ public struct PoolUnitView: View {
 
 					Spacer(minLength: 0)
 
+					TransactionReviewAmountView(amount: viewState.amount, guaranteedAmount: viewState.guaranteedAmount)
```

Comment:
Better to not show the amount in Account details, because it will have potentially a cascading effect - why not do the same for LSU, have Android to update it also.
It is a good idea, but not to be done now.